src/singleton.jl

```diff
@@ -1,9 +1,13 @@
 """
-    SingletonCommsContext()
+    SingletonCommsContext(device = CPU())
 
 A singleton communications context, used for single-process runs.
```

Comment:
add link to possible devices?

Reply:
Addressed in PR https://github.com/CliMA/ClimaComms.jl/pull/33